moveit_setup_assistant/src/widgets/configuration_files_widget.cpp

```diff
@@ -432,6 +444,15 @@ bool ConfigurationFilesWidget::loadGenFiles()
   file.write_on_changes = 0;
   gen_files_.push_back(file);
 
+  // run_benchmark_trajopt.launch --------------------------------------------------------------------------------------
+  file.file_name_ = "run_benchmark_trajopt.launch";
+  file.rel_path_ = config_data_->appendPaths(launch_path, file.file_name_);
+  template_path = config_data_->appendPaths(template_launch_path, file.file_name_);
+  file.description_ = "Launch file for benchmarking TrajOpt planners";
+  file.gen_func_ = boost::bind(&ConfigurationFilesWidget::copyTemplate, this, template_path, _1);
+  file.write_on_changes = 0;
+  gen_files_.push_back(file);
```

Comment:
I'm curious if the moveit benchmarking capabilities support non-OMPL benchmarking... @henningkayser ?

Reply:
Looks like it should be able too, there's nothing specific to OMPL in the package except for [this line](https://github.com/ros-planning/moveit/blob/898cdc8d8df5689319cd462c2c674954c6134604/moveit_ros/benchmarks/src/BenchmarkExecutor.cpp#L527), which just skips non-library plugins.

Reply:
@BryceStevenWilley you're right, since the planning interface is used this should work for other planners as well. I think we could use a similar group configuration pattern like with OMPL, just for solvers instead of algorithms. In that case we would need to adapt the code segment you referenced.